flatten_array.rb

```diff
@@ -0,0 +1,28 @@
+require 'pry'
+
+class FlattenArray
+  def self.flatten(array)
+    flattened_array = []
+
+    array.each do |element|
```

Comment:
Consider using a .map instead of .each. This would not require declaring var in line 5.